app/src/net/fieldwire/models/response/AccountUser.java

```diff
@@ -0,0 +1,33 @@
+package net.fieldwire.models.response;
+
+import java.util.Date;
+
+public class AccountUser extends BaseModel<Integer> {
+    public String firstName;
+    public String lastName;
+    public String accountRole;
+    public String email;
+    public String company;
+    public String phoneNumber;
+    public String photoUrl;
+    public Date currentSignInAt;
+    public int invitedById;
+
+    @Override
+    public String toString() {
+        return "User{" +
+                "firstName='" + firstName + '\'' +
+                ", lastName='" + lastName + '\'' +
+                ", accountRole='" + accountRole + '\'' +
+                ", email='" + email + '\'' +
+                ", company='" + company + '\'' +
+                ", phoneNumber='" + phoneNumber + '\'' +
+                ", photoUrl='" + photoUrl + '\'' +
+                ", currentSignInAt=" + currentSignInAt +
```

Comment:
nit: add single quotes to be consistent with other timestamps

Reply:
Thanks for pointing this out. Looks like the other timestamps were typed as `String` which lead to the quoting. I updated them instead to be `Date` & regenerated the `toString` method for the required classes